polkadot-parachains/src/service.rs

```diff
@@ -676,118 +626,107 @@ where
 		 task_manager,
 		 relay_chain_interface,
 		 transaction_pool,
-		 sync_service,
+		 sync_oracle,
 		 keystore,
-		 force_authoring| {
-			// Encointer customization: This part is different from upstream. We don't use the relay
-			// consensus part from further below, we start with aura consensus directly.
-			//
-			// We do this because our launch-runtime comes with consensus, whereas the upstream
-			// shell runtime does not have that.
```

Comment:
I think we can get rid of this legacy now. agree @clangenb ?
don't think it will ever happen again that we start with a shell which isn't launch-runtime

Reply:
The issue was different, there was extra code from upstream, which we didn't use, and so we removed it, but when I look at the code, it seems that we have harmonized it with upstream now anyhow. So we can remove it.